silver/api/urls.py

```diff
@@ -70,11 +71,13 @@
     url(r'^proformas/(?P<document_pk>[0-9]+)/entries/$',
         views.ProformaEntryCreate.as_view(), name='proforma-entry-create'),
     url(r'^proformas/(?P<document_pk>[0-9]+)/entries/(?P<entry_pk>[0-9]+)/$',
-        views.ProformaEntryUpdateDestroy.as_view(), name='proforma-entry-update'),
+        views.ProformaEntryUpdateDestroy.as_view(
```

Comment:
Same as above, there is no reason to split a call with no arguments on multiple lines.